logstash-core/src/main/java/org/logstash/Valuefier.java

```diff
@@ -145,18 +145,8 @@ private static Object fallbackConvert(final Object o, final Class<?> cls) {
         return converters;
     }
 
-    /**
-     * Converter from either a Java or a Ruby type to a type that both {@link Javafier} and
-     * {@link Rubyfier} are able to convert back their respective types efficiently.
-     */
-    private interface Converter {
+    public interface Converter {
```

Comment:
Should this be explicitly declared as a `@FunctionalInterface`

Reply:
@jakelandis hmm I don't really have an opinion on this. To me, that annotation seems kind of redundant.

Reply:
@original-brownbear - I personally like it explicitly declared, the IDE will also complain louder is you try to add a second method.  

In my 3 minutes of googling, I didn't find any style guides stating one way or the other. 

..so dealers choice :)